packages/graphql-toolbox/src/modules/Main/Main.tsx

```diff
@@ -52,8 +53,28 @@ export const Main = () => {
         );
     }
 
+    const Banner = () => {
+        const handleInterestedInGraphQLaaSClick = () => {
+            window.open("https://forms.gle/uQgai8zaemJz6X4B6", "_blank");
```

Comment:
I remember reading this: https://developer.mozilla.org/en-US/docs/Web/API/Window/open#reuse_existing_windows_and_avoid_target_blank maybe we can provide a name on the target!